docs/getting-started/installation/wpilib.md

```diff
@@ -0,0 +1,8 @@
+# Installing WPILib
+Luckily the installation process for WPILib isn't changed much by the use of Kotlin in your projects, barring a few specific settings when installing that make your life easier.
+
+To install WPILib you can follow the [guide](https://www.jetbrains.com/help/idea/installation-guide.html) contained on the WPILib wiki, with a few special instructions. You can find this guide. Once you reach the page where it asks which install mode you'd like, make sure to select "Everything", and then on the next page select "Skip and don't use VS Code".
```

Comment:
> You can find this guide.

We definitely don't want to come off as snobby. It'll make people not like us, and not like what we're trying to do

Reply:
Oop,s that was a typo lol, I meant to delete that